framework/src/materials/DerivativeSumMaterial.C

```diff
@@ -59,7 +59,9 @@ DerivativeSumMaterialTempl<is_ad>::DerivativeSumMaterialTempl(const InputParamet
     mooseError("Please supply at least one material to sum in DerivativeSumMaterial ", name());
 
   // get prefactor values if not 1.0
-  std::vector<Real> p = this->template getParam<std::vector<Real>>("prefactor");
+  std::vector<Real> p = this->isParamValid("prefactor")
```

Comment:
this is not needed, the param is always valid, it has a default

Reply:
Review addressed